### PR TITLE
feat: desmos umee with tests and specs

### PR DESCRIPTION
**Pull request by henrily-ledger**
### 📝 Description

Add Desmos and umee in cosmos batch 1, functionnal + test

### ❓ Context

- **Impacted projects**: `` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: libs/ledgerjs/packages/cryptoassets/src/abandonseed.ts [hunk 1]
```diff
@@ -65,6 +65,8 @@ const abandonSeedAddresses: Partial<Record<CryptoCurrency["id"], string>> = {
   songbird: "0x0000000000000000000000000000000000000000",
   flare: "0x0000000000000000000000000000000000000000",
   near: "4e7de0a21d8a20f970c86b6edf407906d7ba9e205979c3268270eef80a286e2d",
+  desmos: "cosmos19rl4cm2hmr8afy4kldpxz3fka4jguq0auqdal4",
+  umee: "cosmos19rl4cm2hmr8afy4kldpxz3fka4jguq0auqdal4",
```
Comment (hedi-edelbloute) [inline libs/ledgerjs/packages/cryptoassets/src/abandonseed.ts:69]: What should we do if there is no abandonseed ?

**Review by hedi-edelbloute — APPROVED** [review]

File: libs/ledgerjs/packages/cryptoassets/src/abandonseed.ts (hunk 1 above)
Comment (henrily-ledger) [reply libs/ledgerjs/packages/cryptoassets/src/abandonseed.ts:69]: I use cosmos abandonseed, seems to do the work for osmosis so I keep it

File: libs/ledger-live-common/src/families/cosmos/api/Cosmos.ts [hunk 2]
```diff
@@ -91,10 +91,10 @@ export class CosmosAPI {
   getChainId = async (): Promise<string> => {
     const { data } = await network({
       method: "GET",
-      url: `${this.defaultEndpoint}/node_info`,
+      url: `${this.defaultEndpoint}/cosmos/base/tendermint/v1beta1/node_info`,
```
Comment (haammar-ledger) [inline libs/ledger-live-common/src/families/cosmos/api/Cosmos.ts:94]: @hedi-edelbloute @henrily-ledger `v1beta1` should be replaced with `{this.version$}`

Comment (henrily-ledger) [reply libs/ledger-live-common/src/families/cosmos/api/Cosmos.ts:94]: will do in branch

**Pull request merged**